src/components/grid-aware/BlockQuoteBlock/BlockQuoteBlock.tsx

```diff
@@ -1,30 +1,35 @@
-import PropTypes from "prop-types";
 import { CarouselProvider, Slider, Slide, DotGroup } from "pure-react-carousel";
-import React from "react";
+import * as React from "react";
 
 import "pure-react-carousel/dist/react-carousel.es.css";
 import s from "./BlockQuoteBlock.module.css";
 
-const SlideQuote = ({ quote, attribution }) => (
+type SlideQuoteProps = {
+  quote: string;
+  attribution: string;
+};
+
+const SlideQuote = ({ quote, attribution }: SlideQuoteProps) => (
   <figure className={s.slideQuote}>
     <blockquote className={s.quote}>&ldquo;{quote}&rdquo;</blockquote>
     <figcaption className={s.attribution}>&mdash;{attribution}</figcaption>
   </figure>
 );
 
-SlideQuote.propTypes = {
-  quote: PropTypes.string.isRequired,
-  attribution: PropTypes.string.isRequired,
+type BlockQuoteBlockProps = {
+  quotes: SlideQuoteProps[];
 };
 
-const BlockQuoteBlock = ({ quotes }) => (
+const BlockQuoteBlock = ({ quotes }: BlockQuoteBlockProps) => (
   <div className={s.bleedWrapper}>
     <div className={s.carouselWrapper}>
       <CarouselProvider
         totalSlides={quotes.length}
         isIntrinsicHeight
         isPlaying
-        interval="10000"
+        interval={10000}
+        naturalSlideHeight={0}
+        naturalSlideWidth={0}
```

Comment:
Previously, this component did not require these properties to be passed down as props, but in this TS integration, the linter forced these values to be required. @richardxia

Reply:
Ah, interesting, well it's nice that [`pure-react-carousel`](https://github.com/express-labs/pure-react-carousel) comes with its own official TypeScript types, and that it caught this mistake we made with not passing in required properties. However, I don't think these values of 0 are the correct ones to pass in.

According to their README:

> The only required properties are the orientation, naturalSlideWidth, and naturalSlideHeight. The naturalSlideWidth and naturalSlideHeight are used to create an aspect ratio for each slide. Since the carousel is responsive by default, it will stretch to fill in the width of it's parent container.

That sounds like we should be passing in real values, but the values are not the absolute height and width, but rather relative numbers that allow us to construct the aspect ratio it should be scaled up and down with. Could you figure out the actual aspect ratio we currently have on the site and pick some numbers here that roughly conform to that aspect ratio?

Reply:
Changed width to 45 and height to 10.